### ISSUE TITLE
Expected style "fontSize: 14" to contain units warning from styled components

### ISSUE THREAD

**Issue by rahmanharoon** [torgeadelin/react-native-animated-nav-tab-bar#112]
Here is the warning message which comes from styled components

```
Expected style "fontSize: 14" to contain units  at StyledNativeComponent
```
![Simulator Screen Shot - iPhone 12 mini - 2022-01-05 at 16 27 48](https://user-images.githubusercontent.com/67966429/148207197-4f8485aa-c899-48aa-867c-db874c52d816.png)

**Comment by duongquang1611** [comment]
I added the 'px' suffix after the fontSize in the file node_modules/react-native-animated-nav-tab-bar/dist/lib/index.js (line 193) and the warning was gone.
Here is the code after adding
```
var Label = Styled__default["default"](reactNative.Animated.Text)(templateObject_6 || (templateObject_6 = __makeTemplateObject(["\n\tfontSize: ", "px;\n\tcolor: ", ";\n\tmargin-left: ", "px;\n"], ["\n\tfontSize: ", "px;\n\tcolor: ", ";\n\tmargin-left: ", "px;\n"])), function (p) {
```

**Comment by torgeadelin** [comment]
Fixed, please update to 3.1.8
